src/test/java/seedu/dictionote/testutil/NoteUtil.java

```diff
@@ -0,0 +1,25 @@
+package seedu.dictionote.testutil;
+
+import static seedu.dictionote.logic.parser.CliSyntax.PREFIX_CONTENT;
+
+import seedu.dictionote.logic.commands.AddNoteCommand;
+import seedu.dictionote.model.note.Note;
+
+/**
+ * A utility class for Note.
+ */
+public class NoteUtil {
+
+    /**
+     * Returns an addNote command string for adding the {@code note}.
+     */
+    public static String getAddNoteCommand(Note note) {
+        return AddNoteCommand.COMMAND_WORD + " " + getNoteDetails(note);
+    }
```

Comment:
Shouldn't the JavaDoc for this method contain a tag for the parameter `note`? (i.e., `@param note <description>`)